tweets/urls.py

```diff
@@ -9,4 +9,5 @@
   path('edit/<int:id>', views.editTweet, name='editTweet'),
   path('update/<int:id>', views.updateTweet, name='updateTweet'),
   path('destroy/<int:id>', views.destroyTweet, name='destroyTweet'),
+  path('retweet/<int:id>', views.reTweet, name='reTweet')
```

Comment:
Please explain why did you choose 

```
path('retweet/<int:id>', views.reTweet, name='reTweet')
```

Reply:
because we are going to retweet a tweet. Also there can be other option we can use maybe ```python share/<int:id>```

Reply:
1. change URLS to https://stackoverflow.com/a/67920715/574472
2. Then we will discuss this again

Reply:
You want to change all function base views to class base view?

Reply:
Yes